src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5427,6 +5427,36 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ImplicitCopyInReadOnlyMember_Title" xml:space="preserve">
     <value>Call to non-readonly member from a 'readonly' member results in an implicit copy.</value>
   </data>
+  <data name="ERR_StaticMemberCantBeReadOnly" xml:space="preserve">
+    <value>Static member '{0}' cannot be 'readonly'.</value>
+  </data>
+  <data name="ERR_StaticMemberCantBeReadOnly_Title" xml:space="preserve">
```

Comment:
>ERR_StaticMemberCantBeReadOnly_Title [](start = 14, length = 36)

Are the `ERR_..._Title` members used?

Reply:
I'd like to know that myself. I was copying the convention I saw for the error message resources in this file--maybe when you're filtering the error list in the IDE the title is used to display what each error is generally about?

Reply:
I believe `_Title` members are only needed for warnings, not errors.

---
In reply to: [267959720](https://github.com/dotnet/roslyn/pull/34338#discussion_r267959720) [](ancestors = 267959720)

Reply:
Removed.